### ISSUE TITLE
Add writeWIG() for CBS objects

### ISSUE THREAD

**Issue by HenrikBengtsson** [HenrikBengtsson/PSCBS#21]
Add `writeBed()` method for objects of class `CBS` in order to save segmentation results as [BED](https://genome.ucsc.edu/FAQ/FAQformat.html#format1) files.

BED fields to use:
1. `chrom` - The name of the chromosome (e.g. chr3, chrY, chr2_random) or scaffold (e.g. scaffold10671).
2. `chromStart` - The starting position of the feature in the chromosome or scaffold. The first base in a chromosome is numbered 0.
3. `chromEnd` - The ending position of the feature in the chromosome or scaffold. The `chromEnd` base is not included in the display of the feature. For example, the first 100 bases of a chromosome are defined as `chromStart=0`, `chromEnd=100`, and span the bases numbered 0-99.
The 9 additional optional BED fields are:
4. `name` - Defines the name of the BED line. This label is displayed to the left of the BED line in the Genome Browser window when the track is open to full display mode or directly to the left of the item in pack mode.
5. `score` - A score between 0 and 1000. If the track line `useScore` attribute is set to 1 for this annotation data set, the score value will determine the level of gray in which this feature is displayed (higher numbers = darker gray).

Example of output:
```
track name="CBS Data" description="Circular Binary Segmentation" useScore=1
chr22 1000 5000 cloneA 960 + 1000 5000 0 2 567,488, 0,3512
chr22 2000 6000 cloneB 900 - 2000 6000 0 2 433,399, 0,3601

**Comment by HenrikBengtsson** [comment]
Hmm... should probably use the [Wiggle Track Format (WIG)](http://genome.ucsc.edu/goldenpath/help/wiggle.html) format for this.  See how [writeRegions()](https://github.com/HenrikBengtsson/aroma.core/blob/master/R/CopyNumberSegmentationModel.R#L583-L624) in aroma.core does it.

@grimmermatt, for your needs, will WIG do?

**Comment by grimmermatt** [comment]
Absolutely. Thanks, Henrik!

- Matt On Sun, Sep 06, 2015 at 6:50 PM, Henrik Bengtsson < notifications@github.com [notifications@github.com] > wrote:
Hmm... should probably use the Wiggle Track Format (WIG) [http://genome.ucsc.edu/goldenpath/help/wiggle.html] format for this. See how writeRegions()
[https://github.com/HenrikBengtsson/aroma.core/blob/master/R/CopyNumberSegmentationModel.R#L583-L624] in aroma.core does it.

@grimmermatt [https://github.com/grimmermatt] , for your needs, will WIG do?

—
Reply to this email directly or view it on GitHub
[https://github.com/HenrikBengtsson/PSCBS/issues/21#issuecomment-138154100] .[https://github.com/notifications/beacon/AH-yqPqlr8ZvfI1dNg6wzrWrAHzZ46o7ks5ovOTOgaJpZM4F14DZ.gif]

**Comment by grimmermatt** [comment]
Henrik -

Would you mind if I stopped by to chat about running PSCBS on our exomes? I have a couple questions. Thanks.

Matt Grimmer, PhD
Postdoctoral researcher, Costello Lab
UCSF Helen Diller Family Comprehensive Cancer Center
1450 3rd St, HD-432, San Francisco, CA 94158
415-514-1184 lab, 415-692-4824 cell
http://bit.ly/1tYgc6V

On Sep 8, 2015, at 11:07 PM, Henrik Bengtsson <notifications@github.com<mailto:notifications@github.com>> wrote:


Closed #21<https://github.com/HenrikBengtsson/PSCBS/issues/21>.
